Python/tier2_redundancy_eliminator_bytecodes.c

```diff
@@ -189,6 +177,74 @@ dummy_func(void) {
         }
     }
 
+    op(_TO_BOOL, (value -- res)) {
+        sym_set_type(value, &PyBool_Type);
```

Comment:
This is wrong. `value` is most likely not a boolean.

Reply:
Woops I misunderstood what `PyObject_IsTrue` does. Thanks.!